relay-event-normalization/src/normalize/mod.rs

```diff
@@ -36,6 +36,12 @@ mod contexts;
 pub struct BuiltinMeasurementKey {
     name: String,
     unit: MetricUnit,
+    #[serde(default, skip_serializing_if = "is_false")]
```

Comment:
In my previous review I missed that there is `#[serde(default)]` on the struct, so the one here is redundant. 

```suggestion
    #[serde(skip_serializing_if = "is_false")]
```

Reply:
updated!